packages/yourloops/components/patient/patient-monitoring-prescription.tsx

```diff
@@ -141,6 +145,15 @@ function PatientMonitoringPrescription(props: PatientInfoProps): JSX.Element {
     setMembersMap(membersHasMap);
   };
 
+  useEffect(() => {
+    console.log("action", action);
+    if (defaultTeamId !== undefined) {
+      selectTeam(defaultTeamId);
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why disable the linter instead of adding the dependencies into the array (`[defaultTeamId, selectTeam]`)?

Reply:
because it goes to a infinite loop, my purpose is to set it once like a constructor.

Reply:
`const [selectedTeam, setSelectedTeam] = useState<Team | null>(defaultTeamId !== undefined ? defaultTeamId : null);`

You can do this at line 105 then

Reply:
it does not work because i also need to set  membersHasMap